apps/ident-service/src/main/java/no/nav/registre/testnorge/identservice/provider/IdentServiceController.java

```diff
@@ -0,0 +1,24 @@
+package no.nav.registre.testnorge.identservice.provider;
+
+import lombok.RequiredArgsConstructor;
+import no.nav.registre.testnorge.identservice.service.IdentAppService;
+import no.nav.registre.testnorge.identservice.testdata.response.IdentMedStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/identservice")
```

Comment:
Husk at REST er basert på resurser. Se https://www.codecademy.com/articles/what-is-rest

Altså I dette tilfelle er `ident` resources. Det betyr at vi kan utføre GET ("retrieve a specific resource (by id) or a collection of resources").

Noe som betyr at vi henter resursen ident fra en id (somme også er selve resursen) da forteller REST oss at vi må lage apiet:

`GET /api/v1/identer/{id}` SÅ hvis den finnes retuner 202 OK ident (I dette tilfelle er det akkurat det samme som ident). Hvis den ikke finnes retunerer 404 NOT  FOUNDT. 

På den måten har du laget et REST Api. 

`GET /api/v1/identservice/checkIdentInProd/{ident}`er ikke et gyldig REST API

Reply:
Ja skjønner, skal lese meg opp mer på det 😄